README.md

```diff
@@ -44,24 +44,30 @@ npm run dev
 <br/>
 
 ## Database Setup
-To setup your database, create a folder named `database` in the root directory.
 
-1. Create the database using the following script:
+1. Install PostgreSQL: 
```

Comment:
We don't need this step since `npm install` will automatically take care of this, right?